go.mod

```diff
@@ -9,12 +9,13 @@ require (
 	github.com/onsi/ginkgo/v2 v2.19.0
 	github.com/onsi/gomega v1.33.1
 	github.com/open-policy-agent/cert-controller v0.10.1
-	k8s.io/api v0.29.5
-	k8s.io/apiextensions-apiserver v0.29.5
-	k8s.io/apimachinery v0.29.5
-	k8s.io/client-go v0.29.5
-	k8s.io/code-generator v0.29.5
-	k8s.io/utils v0.0.0-20230726121419-3b25d923346b
+	k8s.io/api v0.31.0-rc.0
```

Comment:
https://github.com/kubernetes-sigs/controller-tools/issues/1017 wait for the 0.31 support.

Reply:
v1.16.1 is published.